src/leihs/core/db/type_conversion.clj

```diff
@@ -39,15 +34,18 @@
   "Transform PGobject containing `json` or `jsonb` value to Clojure
   data."
   [^org.postgresql.util.PGobject v]
-  (let [type  (.getType v)
+  (let [type (.getType v)
         value (.getValue v)]
     (if (#{"jsonb" "json"} type)
       (when value
-        (with-meta (<-json value) {:pgtype type}))
+        (let [json-expected (<-json value)]
+          (if (instance? String json-expected)
+            json-expected
+            (with-meta (<-json value) {:pgtype type}))))
```

Comment:
FYI: had to handle ?wrong-json-str? for `class java.lang.String cannot be cast to class clojure.lang.IObj`

https://github.com/leihs/leihs-procure/blob/d0750d12cc2f7e9fde5c98d3189301cb60ac26a6/_tmp/errors/jsonb-string-to-object-cast-exception.txt

Reply:
we most probably need a type cast on the schema.edn level, similar to UUID (see my comment for `utils.clj` and the `my-cast` function)

Reply:
FYI: I could remove my-cast and introduced **:ID (UUID) :Int** and **:Timestamptz** in schema.edn